src/controllers/playback/video/index.js

```diff
@@ -435,6 +436,9 @@ import 'css!assets/css/videoosd';
                 const elem = osdBottomElement;
                 clearHideAnimationEventListeners(elem);
                 elem.classList.add('videoOsdBottom-hidden');
+
+                mouseManager.hideCursor();
+
```

Comment:
Move this to https://github.com/jellyfin/jellyfin-web/blob/d7526d955d67f26bedf74d40b51408e683686cce/src/controllers/playback/video/index.js#L368-L371
In this case, cursor will be hidden at the same time as header when UpNext dialog is displayed.